homeassistant/components/plaato/sensor.py

```diff
@@ -72,17 +72,11 @@ async def _async_update_from_webhook(device_id, sensor_data: PlaatoDevice):
 class PlaatoSensor(PlaatoEntity, SensorEntity):
     """Representation of a Plaato Sensor."""
 
-    @property
-    def device_class(self) -> SensorDeviceClass | None:
-        """Return the class of this device, from SensorDeviceClass."""
-        if (
-            self._coordinator is not None
-            and self._sensor_type == PlaatoKeg.Pins.TEMPERATURE
-        ):
-            return SensorDeviceClass.TEMPERATURE
```

Comment:
It does not seem it is need to check for the coordinator to be set here as it is not referenced.